cfme/tests/infrastructure/test_config_management_rest.py

```diff
@@ -230,3 +231,69 @@ def test_authentications_options(self, appliance, config_manager):
         collection = appliance.rest_api.collections.authentications
         assert 'credential_types' in collection.options()['data']
         assert_response(appliance)
+
+
+@pytest.fixture(scope="module")
+def config_manager_rest(config_manager_obj):
+    """Creates provider using REST API."""
+    config_manager_obj.create_rest()
+    assert_response(config_manager_obj.appliance)
+    rest_entity = config_manager_obj.rest_api_entity
+    rest_entity.reload()
+
+    yield rest_entity
+
+    if rest_entity.exists:
+        rest_entity.action.delete()
+        rest_entity.wait_not_exists()
+
+
+@pytest.mark.tier(3)
+@test_requirements.rest
+def test_config_manager_create(config_manager_rest):
+    """
+    Polarion:
+        assignee: pvala
+        casecomponent: Rest
+        caseimportance: medium
+        initialEstimate: 1/10h
+
+    Bugzilla:
+        1621888
+    """
+    assert "?provider_class=provider" in config_manager_rest.href
+    assert "::Provider" in config_manager_rest.type
+
+
+@pytest.mark.tier(3)
+@test_requirements.rest
+def test_config_manager_edit(request, config_manager_rest):
```

Comment:
I think you'll end up with a duplicate test name here from test_config_management.py.  I think your test function names here should include `_rest`